product_docs/docs/pgd/5/proxy/index.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "EDB Postgres Distributed Proxy"
+navTitle: "PGD Proxy"
+---
+
+EDB Postgres Distributed Proxy is a daemon that acts as an abstraction layer between the client application and Postgres. It interfaces with the BDR consensus mechanism to obtain the identity of the current write leader node and redirects traffic to that node.
+
+There is always at least one global group and one data group in the PGD cluster. BDR elects the write leader for each data group which has the `enable_proxy_routing` option set to true. Proxy can be attached to a global group or data group. There could be multiple proxies attached to each group.
+
+PGD Proxy is a TCP layer 4 proxy. It can't encode/decode the PostgreSQL Wire Protocol. Although this may change in future releases.
```

Comment:
remove this - `It can't encode/decode the PostgreSQL Wire Protocol. Although this may change in future releases.`